libs/doctrine-retry-bundle/README.md

```diff
@@ -0,0 +1,11 @@
+# Doctrine Retry Bundle
+
+## Installation
+```bash
+composer require keboola/doctrine-retry-bundle
+```
+
```

Comment:
Vim ze to je uz muj kostlivec, kdyz jsem to puvodne implementoval, ale hodil by se sem jeste example, ze aktivovat to jde jednoduse nastavenim `x_connect_restriesL: number` do `options` DBALu https://github.com/keboola/job-queue/blob/2b71bacda33f49be2ec33d4fe92295e91dafea7e/apps/internal-api/config/packages/doctrine.yaml#L6

```
doctrine:
    dbal:
        ...
        options:
            x_connect_retries: 3
```

Reply:
fixed